docs/source/docker/examples/datasets_in_the_wild.rst

```diff
@@ -112,11 +112,19 @@ selects a diverse set of frames:
           "enable_training": False,
           "pretagging": False,
           "pretagging_debug": False,
-          "method": "coreset",
-          "stopping_condition": {
-              "n_samples": 99,
-              "min_distance": -1
-          }
+      },
+      selection_config = {
+          "nSamples": 99,
+          "strategies": [
+              {
+                  "input": {
+                      "type": "EMBEDDINGS"
+                  },
+                  "strategy": {
+                      "type": "DIVERSIFY"
+                  }
+              }
+          ]
```

Comment:
The sentence on line 132 needs an update as well: "To perform a random selection we can simply replace "coreset" with "random" as our selection method. Note that Coreset is the default method."